skbio/sequence/tests/test_grammared_sequence.py

```diff
@@ -0,0 +1,604 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+import six
+
+from unittest import TestCase, main
+
+import numpy as np
+import numpy.testing as npt
+
+from skbio.sequence import GrammaredSequence
+from skbio.sequence._grammared_sequence import GrammaredSequenceException
+from skbio.util._decorator import classproperty
```

Comment:
Interesting that checklist.py didn't catch this... can you change to `from skbio.util import classproperty`?